Ham/DabraLibanosHamGG1/msDesc.xml

```diff
@@ -0,0 +1,1782 @@
+        </handNote>
+        <handNote script="Ethiopic" xml:id="h13" corresp="#a25">
+          <locus from="12va" to="12vb"/>
+        </handNote>
+      </handDesc>
+    <additions>
+      <list>
+        <item xml:id="a26">
+          <locus from="11r" to="11r"/>
+          <!--          please Nafisa could you tell me what these are and if consistent with the rest of the text on this page?-->
+          <note>texts on upper margin</note>
+        </item>
+        <item xml:id="a23">
+          <locus from="11ra1" to="11rb17"/>
+          <q xml:lang="gez"> </q>
+          <note>[4.34]: (I f. 11ra-b) [non reperibile in CCR], according to AB.</note>
+        </item>
+        <item xml:id="a24">
+          <locus from="11va1" to="11vb6"/>
+          <desc type="GuestText"><title ref="LIT6306CCR1"/></desc>
+          <q xml:lang="gez">
+            <ab>ዘወሐቡ፡ ንጉሥ፡ ገብረ፡ መስቀል፡ ለአባ፡ መጣዕ፡ ሰርኤ፡ ለጎ፡ ኦና፡ ሰያባ፡ ወንከብ፡ ስዓጋ፡ ማርያላ፡ ገማዕ፡ ዜና፡
+              ኵዶ፡ አዲ፡ ሕልሙ፡ ጽዳዕ፡ አሲራ፡ ለባሲሮ፡ <sic>ገዳሙሙ፡ </sic>ለመነኮሳት፡ ቤተ፡ ሰማዕት፡ አናንያ፡ ወአዛርያ፡
+              ወሚሳኤል፡ ፭ቱ፡ አዲ፡ በጊኤን፡ ፃዕዳ፡ ብአቲ፡ አዲ፡ አይኬማ፡ አዲ፡ አይሸሑት፡ ምድፉዕ፡ ዋልታ፡ አኦመ፡ ረጌንያ፡ ሰማዝ፡
+              ጉናጉና፡ ቀዳማዊ፡ <term type="act">ጉልቱ፡ </term>ለአባ፡ መጣዕ፡ በሕይወቱ፡ በአኰቴተ፡ አብ፡ ወወልድ፡
+              ወመንፈስ፡ ቅዱስ፡ አነ፡ ገብረ፡ መስቀል፡ ዘንተ፡ ጉልተ፡ ዘወሐብኩ፡ አዕሚርየ፡ ከመ፡ ይሐልፍ፡ ኵ<cb/> ሉ፡ ብዕል፡
+              ኵlb ዳወይባ፡ አወገር፡ ፭ቱ፡ አዲ፡ ምስለ፡ ምድሮሙ፡ ሐፉሁ፡ ወመስኖሁ፡ ደረቃ፡ <add place="above"
+                >በ</add>ሲሬ፡ </ab>
+          </q>
+          <note>[4.36] = CCR 1: (I f. 11va-b), according to AB.</note>
+        </item>
+        <item xml:id="a27">
+          <locus from="12va1" to="12va8"/>
+          <desc type="CommemorativeNote"/>
+        </item>
+        <item xml:id="a25">
+          <locus from="12va9" to="12vb20"/>
+          <desc type="GuestText"><title ref="LIT6307CCR2"/></desc>
+        </item>
+      </list>
+    </additions>
+    </physDesc>
+    <history>
+      <origin>Conti Rossini proposed that this part of the manuscript is of the 
+        <origDate resp="#evangeloDoro" notBefore="1601" notAfter="1700">XVII century</origDate> 
+        in <bibl><ptr target="bm:ContiRossini1901Evangelodoro"/><citedRange unit="page">181</citedRange></bibl>.</origin>
+    </history>
+    
+    <msPart xml:id="p2" corresp="#quire2">
+
+      <msIdentifier>
+        <repository ref="INS0346DL"/>
+      </msIdentifier>
+
+      <msContents>
+        <!--      matchings from Bausi's list -->
+
+        <msItem xml:id="p2_i1">
+          <locus from="3ra1" to="4vb14"/>
+          <title ref="LIT6329CCR22"/>
+          <incipit><subst>
+            <del rend="erasure"/>
+            <add place="overstrike">አጽሐ</add>
+          </subst> ፍኩ፡ አነ፡ <persName ref="PRS1854Amdase">ዐምደ፡ ጽዮን፡ ንጉሥ። </persName>
+            በ፭፻፲ወ፪<sic>ዐመት፡ </sic>
+          </incipit>
+          <explicit>ውጉዘ፡ ለይኩን፡ በዝዓለም፡ ወበዘይመጽእ፡ ዓለም፡ ዘኢይትፈታሕ፡ በጾም፡ ወበ<del rend="erasure"/>ጸ<del
+            rend="erasure"/>
+            <pb/><cb/>ሎት፡ በስኢል፡ ወበስጊድ፡ ለዓለመ፡ ዓለም። <lb/><space/>
```

Comment:
I thik CCR 22 can be treated as one document ending up with a prayer, which is its part